deformable_ompl/SimplifyPathQuery[] queries
---
deformable_ompl/SimplifyPathResult[] results
